src/components/dialogs/NameRequestErrorDialog.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-dialog v-model="dialog" persistent :attach="attach" max-width="60%">
+  <v-dialog v-model="dialog" persistent :attach="attach" max-width="600px">
```

Comment:
Do you think this should be made into a shared component?
(I don't want to drag down this PR so, if yes, this can be done in a future PR.)

Reply:
Yea! That sounds fun, i really want to give that a go, sooner than later